languages/elixir/exercises/concept/bitstrings/.docs/hints.md

```diff
@@ -0,0 +1,43 @@
+## General
+
+- Use `?` to work with the character [codepoints][codepoint].
+- `\s` can be used to represent a space.
+- Use [integer binary notation][integer-literal] for working with the codes.
+- Try to use the tail call recursion strategy.
+
+## 1. Encode nucleic acid to binary value
+
+- This function needs to map one integer to another.
+- Making use of multiple clause functions may make this easier by breaking it down.
+
+## 2. Decode the binary value to the nucleic acid
+
+- This function is the opposite of part 1's function.
+- Making use of multiple clause functions may make this easier by breaking it down.
+
+## 3. Encode a DNA charlist
+
+- Create a recursive function which takes a codepoint from the charlist and recursively builds the bitstring result.
+
+- Remember, a [charlist][charlist] is a list of [integer codepoints][codepoint].
+- You can get the first item from a list using the [`hd/1`][hd] function.
+- You can get the remaining items from a list using the [`tl/1`][tl] function.
```

Comment:
I think Erik would say to avoid revealing function names in the text and say something like "by using a [built-in function]".